src/config/getconfig.cc

```diff
@@ -11,8 +11,8 @@ using namespace std;
 using namespace google::protobuf::util;
 using namespace authservice::config;
 
-// TODO: s/namespace transparent_auth/namespace authservice/g
-namespace transparent_auth {
+// TODO: s/namespace authservice/namespace authservice/g
```

Comment:
Can you remove this TODO. You've just made the changes it describes are required.

Reply:
No problem! Should have just deleted it as I was going through the code.